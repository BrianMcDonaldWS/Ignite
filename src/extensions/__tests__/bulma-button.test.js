import MarkdownIt from 'markdown-it';
import button from '../bulma-button';

const md = new MarkdownIt().use(button);

test('bulma - button', () => {
  expect(
    md.render(`
::: button is-large is-link is-bold is-inverted is-outlined
[Get Started :tada:](pages/GettingStarted.md)
:::
  `)
  ).toMatchSnapshot();
});
