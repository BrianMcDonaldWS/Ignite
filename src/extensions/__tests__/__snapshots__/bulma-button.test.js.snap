// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bulma - button 1`] = `
"
          <button class=\\"button  is-large is-link is-bold is-inverted is-outlined\\">
      <p><a href=\\"pages/GettingStarted.md\\">Get Started :tada:</a></p>

      </button>
    "
`;
